docs/html/topics/repeatable-installs.md

```diff
@@ -0,0 +1,98 @@
+# Repeatable Installs
+
+pip can be used to achieve various levels of repeatable environments. This page
+walks through increasingly stricter definitions of what "repeatable" means.
+
+## Pinning the package versions
+
+Pinning package versions of your dependencies in the requirements file
+protects you from bugs or incompatibilities in newly released versions:
+
+```
+SomePackage == 1.2.3
+DependencyOfSomePackage == 4.5.6
+```
+
+```{note}
+Pinning refers to using the `==` operator to require the package to be a
+specific version.
```

Comment:
Do we need to mention the even stricter `===`?

Reply:
Nah, that mostly exists for compatibility with legacy versions; which we don't really have a lot of anymore.

Reply:
It’s also useful for excluding local versions, but okay.